src/app/utils/dom-utils.ts

```diff
@@ -0,0 +1,7 @@
+export const getElementViewportOffset = (el: HTMLElement, elementHeight = el.clientHeight) => {
+  const windowHeight = window.outerHeight,
+    elRect = el.getBoundingClientRect(),
+    top = elRect.top,
+    bottom = elRect.bottom
+  return Math.max(0, top > 0 ? Math.min(elementHeight, windowHeight - top) : Math.min(bottom, windowHeight))
+}
```

Comment:
I overall don't think this feature is worth the added complexity of https://github.com/oasisprotocol/explorer/pull/482 + this.
Just show the picker over the topbar :shrug: -10% of project's complexity

Reply:
I agree on complexity. One simple solution would be to always scroll to the top, before showing the mobile drawer. Because banners are causing the Picker to look like it is misaligned.